src/constants.mjs

```diff
@@ -328,3 +328,10 @@ export const DOC_TYPES_MAPPING_OTHER = {
   Response: `${DOC_MDN_BASE_URL}/API/Response`,
   Request: `${DOC_MDN_BASE_URL}/API/Request`,
 };
+
+// Mapping that determines which generators are run on the target modes used in the CLI
+export const CLI_TARGET_MAPPING = {
```

Comment:
I'm not sure if we need this kind of mapping, we can also use generators directly? 🤔

Reply:
I'm not sure this mapping is needed, it should be literally as it says, since jsonSimple !== json; We will have both a HTML and JSON generators, so it's incorrect to call the jsonSimple and legacyHtml as json and HTML.

BTW could we rename on the source "jsonSimple" to "json-simple" and "legacyHtml" to "legacy-html" ?